source/brailleDisplayDrivers/albatross.py

```diff
@@ -0,0 +1,822 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2021 NV Access Limited, Burman's Computer and Education Ltd.
+
+import ctypes
+import serial
+import time
+
+from collections import deque
+from ctypes import byref
+from ctypes.wintypes import DWORD
+from logHandler import log
+from serial.win32 import ERROR_IO_PENDING, EV_RXCHAR
+from serial.win32 import PURGE_RXABORT, PURGE_TXABORT
+from serial.win32 import PURGE_RXCLEAR, PURGE_TXCLEAR
+from serial.win32 import GetLastError, PurgeComm, SetCommMask
+from threading import Event, Lock, Timer, Thread
+from typing import List, Optional, Tuple

```

Comment:
For readability, these should be split as multi-line imports, e.g:

```suggestion
...
from typing import (
	List,
	Optional,
	Tuple,
)
```